docs/buildspecs/spack.rst

```diff
@@ -151,6 +151,30 @@ will create an environment **manifest_example** using the manifest file that we
 
     .. program-output:: cat buildtest_tutorial_examples/spack/inspect/env_create_manifest.txt
 
+Deactivate Spack Environment
+----------------------------
+
+We added a new feature to deactivate environments using the key word ``deactivate`` which is boolean. If ``deactivate: true`` is set, we can deactivate spack environment prior to activating it via ``spack env deactivate``
```

Comment:
In first sentence do the following

1. Add `type` after boolean so  `which is a boolean type`. 
2. Remove `s` from `environments`
In 2nd sentence say the following

If ``deactivate: true`` is set, then we will deactivate the spack environment by running `spack env deactivate`.